aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -227,16 +194,6 @@ To reduce the logging output, set "Logging:LogLevel:Microsoft:Error" in the *app
 
 [!code-json[](dependency-injection/samples/3.x/DependencyInjectionSample/appsettings.Development.json?highlight=7)]
 
-## Call services from main
```

Comment:
@halter73 nice to get rid of this section. I like the new minimal hosting model.

Reply:
I added this back and marked the code that demonstrates this.